shared/common-adapters/index.d.ts

```diff
@@ -42,12 +42,7 @@ export {default as FloatingPicker} from './floating-picker'
 export {usePopup} from './use-popup'
 export {default as OverlayParentHOC} from './overlay/parent-hoc'
 export {default as HeaderHoc, HeaderHocHeader, HeaderHocWrapper} from './header-hoc'
-export {
-  default as HeaderOrPopup,
-  HeaderOrPopupWithHeader,
-  PopupWrapper,
-  PopupWithHeaderWrapper,
-} from './header-or-popup'
+export {PopupWrapper} from './header-or-popup'
```

Comment:
maybe rename this file?

Reply:
i'll do that kind of stuff when i'm done removing